card/mod/basic_formats/set/all/json.rb

```diff
@@ -38,11 +39,7 @@ def show view, args
   end
 
   def name_search query_attribute: :complete, limit: AUTOCOMPLETE_LIMIT
-    # context is "" so that term will not be interpreted in the context
-    # of search card name.  However, this can break searches where the
-    # search card name is required (eg found_by)
```

Comment:
I removed `context: ""` because I don't think it can influence the interpretation of the search term any more. There is now code in query/interpretation.rb that checks that a name matches the pattern `/\b_/` (eg `_left`, `_right`) before absolutizing.  So the search term won't get absolutized unless it's relative, which is already a weird case.

This means we can use, eg, `Company+*type+by_name` as an options card on autocomplete.